feature/1-ci-cd-implementation

matlin100 opened the pull request.

matlin100 committed:
Add site web manifest to metadata in layout.tsx

matlin100 committed:
Update layout.tsx to include site web manifest metadata for improved PWA support

matlin100 committed:
.ymal cy cd

matlin100 committed:
Remove unnecessary blank line in print-project-ids.yml workflow file

hodayakaro123 commented on .github/workflows/print-project-ids.yml:
```diff
@@ -0,0 +1,33 @@
+name: Print Project IDs (one-off)
+on:
+  workflow_dispatch:
```
I looked into it and found that the workflow needs additional permissions to access the project data.  
Please make sure to include after on: 
permissions:
  contents: read
  projects: read

matlin100 committed:
Add GitHub Actions workflow to print project IDs

- Introduced a new workflow in `print-project-ids.yml` that retrieves and logs project IDs and their details from a specified GitHub organization and project.
- The workflow is triggered manually and uses GitHub's GraphQL API to fetch project information.
